values/mainnet/network/space06.yaml

```diff
@@ -11,10 +11,15 @@ init:
    clearRestart:
      enabled: false
 
-Lotus:
+lotus:
```

Comment:
Are you sure we need to have this change?
I think we need to revert or make additional changes